spring-cloud-dataflow-server/pom.xml

```diff
@@ -37,7 +34,7 @@
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-dataflow-server</artifactId>
-			<version>${dataflow.version}</version>
```

Comment:
This was like this for a particular reason. The reason being that when this is used in SCDF-PRO the project.version is resolves as the PRO version number - not the SCDF version number. Please revert.